PayKids/presentation/src/main/java/com/paykids/presentation/view/mypage/PolicyFragment.kt

```
@@ -0,0 +1,32 @@
+package com.paykids.presentation.view.mypage
+
+import com.paykids.presentation.base.BaseFragment
+import com.paykids.presentation.databinding.FragmentPolicyBinding
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class PolicyFragment : BaseFragment<FragmentPolicyBinding>() {
+    override fun initView() {
+
+    }
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)